app/web/src/components/Navigation/Navigation.tsx

```diff
@@ -259,49 +264,59 @@ export default function Navigation() {
   const menuItems = (
     <div>
       {menuDropDown(data).map(
-        ({ name, title, notificationCount, route, target }) => {
-          if (title)
-            return (
-              <MenuItem classes={{ root: classes.menuTitle }}>{name}</MenuItem>
-            );
-
-          const nameElement = <Typography noWrap>{name}</Typography>;
+        ({
+          name,
+          notificationCount,
+          route,
+          target,
+          externalLink,
+          hasBottomDivider,
+        }) => {
           const hasNotification =
-            notificationCount !== undefined && notificationCount > 0;
+            notificationCount !== undefined && notificationCount > -1;
 
-          return (
-            <MenuItem key={name}>
-              <a
-                href={route}
-                target={target || "_self"}
-                className={`${classes.menuItem} ${
-                  hasNotification && classes.menuItemMessage
-                }`}
-              >
-                {hasNotification ? (
-                  <Badge
-                    color="primary"
-                    variant="dot"
-                    classes={{ badge: classes.badge }}
-                  >
-                    {nameElement}
-                  </Badge>
-                ) : (
-                  nameElement
-                )}
+          const linkContent = (
+            <MenuItem
+              classes={{
+                root: classNames(classes.menuItem, {
+                  [classes.menuItemMessage]: hasNotification,
+                  [classes.menuItemDivider]: hasBottomDivider,
+                }),
+              }}
+            >
+              {hasNotification ? (
+                <Badge
+                  color="primary"
+                  variant="dot"
+                  classes={{ badge: classes.badge }}
```

Comment:
use className and target the badge with `"& .MuiBadge-badge": {...}` or whatever it is